pyproject.toml

```diff
@@ -18,7 +18,7 @@ keywords = [
     "ecdsa",
 ]
 classifiers = [
-    "Development Status :: 4 - Beta",
+    "Development Status :: 5 - Production/Stable",
```

Comment:
Moving to Prod / Stable makes sense given our user base seems quite stable.  Any concerns from anyone?  I just wanted to flag this for further comment...

Reply:
Oh right, I didn't mention that: I figured this was just forgotten since 1.0 in May 2024 was supposed to mark stability...